src/Billing/EDI270.php

```diff
@@ -325,17 +320,11 @@ public static function createIEA($row, $X12info, $segTer, $compEleSep)
 
     public static function translateRelationship($relationship)
     {
-        switch ($relationship) {
-            case "spouse":
-                return "01";
-                break;
-            case "child":
-                return "19";
-                break;
-            case "self":
-            default:
-                return "S";
-        }
+        return match ($relationship) {
+            "spouse" => "01",
+            "child" => "19",
+            default => "S",
```

Comment:
this seems odd that missing `self`. does the `self` above act like default since no break?